c7n/tags.py

```diff
@@ -511,9 +527,42 @@ def process(self, resources):
         return resources
 
 
+TZ_ALIASES = {
+    'pdt': 'America/Los_Angeles',
```

Comment:
instead of copying and pasting this,  just use/reference the one in filters/offhours.py